server/fleet/microsoft_mdm.go

```diff
@@ -0,0 +1,3 @@
+package fleet
+
+// Placeholder for Fleet service related functions
```

Comment:
I think the pattern is for the content of `server/fleet/microsoft_mdm_types.go` to go here, and any service functions/implementation details would go  in `server/service/mdm.go` or `server/service/microsoft_mdm.go`

Even the request/response types (eg: `SoapRequest`) might end up in `server/service`, as we generally put them alongside the endpoint handler (notice how this tends to happen naturally as you're defining methods for those structs in `server/service/microsoft_mdm.go`)

I think it's okay for all of the types to be in `server/fleet` for now until the dust settles, but I would move them to this file.

Reply:
Thanks for the suggestion! I moved the types in `server/fleet/microsoft_mdm_types.go` to `server/fleet/microsoft_mdm.go` to follow the implementation pattern.

Reply:
Moving `SoapRequest`/`SoapResponse` was a bit more complicated. I'll try to get this done on the upcoming PRs

Reply:
thank you! I wouldn't worry about those for now, we can revisit as we understand more